object_detection/tf_serving_cpu.md

```diff
@@ -17,12 +17,14 @@ MODEL_COMPONENT=pets-model
 MODEL_PATH=/mnt/exported_graphs/saved_model
 MODEL_STORAGE_TYPE=nfs
 NFS_PVC_NAME=pets-pvc
+TF_SERVING_VERSION=`kubectl -n kubeflow exec -ti tf-training-job-master-0 -- python -c 'from __future__ import print_function;import tensorflow;print(tensorflow.__version__, end="")'`
```

Comment:
Is there a cleaner way to handle this? Why do we wait until runtime to get the TF version? Shouldn't this be determined by the code that gets checked in?
https://github.com/kubeflow/examples/blob/master/object_detection/docker/Dockerfile.training#L45

So can't we just have the code pin consistent version of TF across the example?

Reply:
Thanks for your comment. Agree with you. Since default OBJ_DETECTION_IMAGE tensorflow version is 1.10.0, we pin consistent version 1.10.0 of TF across the example.
Updated.